openupgrade_scripts/scripts/mail/15.0.1.5/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,266 @@
+
+mail         / mail.followers           / channel_id (many2one)         : DEL relation: mail.channel
+mail         / mail.followers           / email (char)                  : not a function anymore
+mail         / mail.followers           / email (char)                  : now related
+mail         / mail.followers           / is_active (boolean)           : not a function anymore
+mail         / mail.followers           / is_active (boolean)           : now related
+mail         / mail.followers           / name (char)                   : not a function anymore
+mail         / mail.followers           / name (char)                   : now related
+# NOTHING TO DO: handle by ORM
+
+mail         / mail.followers           / partner_id (many2one)         : now required
+# DONE: pre-migration: delete all channel follower
+
+mail         / mail.guest               / access_token (char)           : NEW required, req_default: function, hasdefault
+mail         / mail.guest               / channel_ids (many2many)       : NEW relation: mail.channel
+mail         / mail.guest               / country_id (many2one)         : NEW relation: res.country
+mail         / mail.guest               / image_1024 (binary)           : NEW attachment: True, isrelated: related, stored
+mail         / mail.guest               / image_128 (binary)            : NEW attachment: True, isrelated: related, stored
+mail         / mail.guest               / image_1920 (binary)           : NEW attachment: True
+mail         / mail.guest               / image_256 (binary)            : NEW attachment: True, isrelated: related, stored
+mail         / mail.guest               / image_512 (binary)            : NEW attachment: True, isrelated: related, stored
+mail         / mail.guest               / lang (selection)              : NEW selection_keys: function
+mail         / mail.guest               / name (char)                   : NEW required
+mail         / mail.guest               / timezone (selection)          : NEW selection_keys: function
+# NOTHING TO DO: new feature
+
+mail         / mail.ice.server          / credential (char)             : NEW
+mail         / mail.ice.server          / server_type (selection)       : NEW required, selection_keys: ['stun', 'turn'], req_default: function, hasdefault
+mail         / mail.ice.server          / uri (char)                    : NEW required
+mail         / mail.ice.server          / username (char)               : NEW
+# NOTHING TO DO: new feature call audio and video
+
+mail         / mail.mail                / failure_type (selection)      : NEW selection_keys: ['mail_bl', 'mail_dup', 'mail_email_invalid', 'mail_email_missing', 'mail_optout', 'mail_smtp', 'unknown']
+# NOTHING DO DO
+
+mail         / mail.mail                / is_notification (boolean)     : NEW
+mail         / mail.mail                / notification (boolean)        : DEL
+# DONE: pre-migration: rename fields notification -> is_notification
```

Comment:
I don't see this one in the pre-migration. Was it left out by mistake?

Reply:
ohh, it's my mistake